### ISSUE TITLE
Fullscreen YouTube videos don't escape the browser window

### ISSUE THREAD

**Issue by ColemanGariety** [spolu/breach_core#149]
If you fullscreen a YouTube video, it'll just expand to the full size of the browser window and be covered by the page's UI elements. Something is wrong.

![screen shot 2014-07-16 at 12 49 49 pm](https://cloud.githubusercontent.com/assets/1356828/3604635/5e916194-0d22-11e4-82c6-e6bfb4fab096.png)

**Comment by brayniverse** [comment]
I can confirm that this happens for me too.

**Comment by GuiHarrison** [comment]
Yep. Same here. It also freezes a lot, frequently only showing a few frames of the video. I'm currently using HTML5 YouTube.

**Comment by t-lutz** [comment]
Are you all using OSX? Did you build from source or are you using binaries?

**Comment by ColemanGariety** [comment]
I'm using OS X and binaries.

**Comment by GuiHarrison** [comment]
I was using Ubuntu.
